recipe-server/client/control/components/recipes/HistoryItem.js

```diff
@@ -0,0 +1,256 @@
+import { Alert, Icon, Popover, Tag, Timeline } from 'antd';
+import autobind from 'autobind-decorator';
+import { Map } from 'immutable';
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'redux-little-router';
+import moment from 'moment';
```

Comment:
Looks like it will be available in eslint _some_ day: https://github.com/benmosher/eslint-plugin-import/pull/629

But for now, I will just be more mindful. Sorry, usually I'm better about that stuff.